__tests__/api/chat/send-signed-message.test.js

```diff
@@ -63,26 +63,4 @@ describe('POST /chat/send-signed-message', () => {
       })
     );
   });
-
-  test('should return 403 error validation if channelType is not chat or group', async () => {
-    const response = await supertest(app)
-      .post('/api/v1/chat/send-signed-message')
-      .set({Authorization: `Bearer token`})
-      .send({
-        message: 'Hello World',
-        channelType: 3
-      })
-      .expect(403);
-
-    expect(response.body).toEqual(
-      expect.objectContaining({
-        message: 'Error validation',
-        error: expect.arrayContaining([
-          expect.objectContaining({
-            message: expect.any(String)
-          })
-        ])
-      })
-    );
-  });
 });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the test case that checks for a 403 error when `channelType` is neither 'chat' nor 'group' could potentially lead to less coverage in your tests. If this validation is still relevant, consider adding it back to ensure that invalid `channelType` values are properly handled.
```diff
+  test('should return 403 error validation if channelType is not chat or group', async () => {
+    const response = await supertest(app)
+      .post('/api/v1/chat/send-signed-message')
+      .set({Authorization: `Bearer token`})
+      .send({
+        message: 'Hello World',
+        channelType: 3
+      })
+      .expect(403);
+
+    expect(response.body).toEqual(
+      expect.objectContaining({
+        message: 'Error validation',
+        error: expect.arrayContaining([
+          expect.objectContaining({
+            message: expect.any(String)
+          })
+        ])
+      })
+    );
+  });
```
If the validation is no longer necessary due to changes in the application logic, then the removal of this test case is justified. However, please ensure that there are other test cases covering the new valid `channelType` values.


<!-- This is an auto-generated comment by OSS CodeRabbit -->